### DISCUSSION TITLE
Why do all the faces look the same? How to turn off this feature on fooocus?

### DISCUSSION THREAD

**Discussion by D-Mad** [lllyasviel/Fooocus#3030] [Q&A] [answered]
![image](https://github.com/lllyasviel/Fooocus/assets/8207507/70db3855-a441-47b4-a11e-e4a5cd5a0e1b)

**Comment by Rogala** [comment]
Maybe you have the Random checkbox unchecked, then all models will be maximally the same.
![image](https://github.com/lllyasviel/Fooocus/assets/7437582/6c6c1c34-c997-495f-9dea-d75a1938e5f0)

**Comment by mashb1t** [comment, marked as answer]
@D-Mad you can add a wildcard with face descriptions and use it by adding `__faces__` to your prompt if you call it faces.txt.
The faces look the same because the default "woman" face is used which the model (probably you're using juggernaut) has been trained on.
